junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestIntegrationTests.java

```diff
@@ -695,6 +695,12 @@ void testWithAggregator(@AggregateWith(StringAggregator.class) String concatenat
 			fail("concatenation: " + concatenation);
 		}
 
+		@ParameterizedTest
+		@CsvSource(value = { " ab,  cd", "ef, gh" }, ignoreTrailingAndLeadingWhitespace = false)
+		void testCsvSourceWithIgnoreTrailingAndLeadingWhitespace(String argument) {
+			fail(argument);
+		}
+
```

Comment:
The test that uses this method is missing, isn't it?

Reply:
These need corresponding test cases. See `executesWithCsvSource` in the enclosing class for an example.

Reply:
Yes, please see the commits. I accidently added the PR for another issue. have reverted it. Thanks for the review.